tests/integration/lxd/test_launcher.py

```diff
@@ -116,23 +116,18 @@ def get_instance_and_base_instance(get_base_instance, instance_name):
             base_instance.delete()
 
 
+# exclude 23.04 (lunar) because it is a daily, not a supported release
 @pytest.mark.parametrize(
-    "alias,image_name",
-    [
-        (bases.BuilddBaseAlias.XENIAL, "16.04"),
-        (bases.BuilddBaseAlias.BIONIC, "18.04"),
-        (bases.BuilddBaseAlias.FOCAL, "20.04"),
-        (bases.BuilddBaseAlias.JAMMY, "22.04"),
-    ],
+    "alias", list(set(bases.BuilddBaseAlias) - {bases.BuilddBaseAlias.LUNAR})
```

Comment:
```suggestion
    "alias", set(bases.BuilddBaseAlias) - {bases.BuilddBaseAlias.LUNAR}
```

There's no advantage to turning this back into a list since it'll keep the set order. If you want it ordered you could do:

```
    "alias", [base for base in bases.BuildBaseAlias if base not in {bases.BuilddBaseAlias.LUNAR}]
```

Reply:
Ah, thanks!  I still find myself forgetting how interchangeable python data structures are.